static/app/components/events/interfaces/crashContent/exception/stackTrace.spec.tsx

```diff
@@ -0,0 +1,187 @@
+    entries: [],
+    crashFile: {
+      sha1: 'sha1',
+      name: 'name.dmp',
+      dateCreated: '2019-05-21T18:01:48.762Z',
+      headers: {'Content-Type': 'application/octet-stream'},
+      id: '12345',
+      size: 123456,
+      type: 'event.minidump',
+    },
+    culprit: '',
+    dateCreated: '2019-05-21T18:00:23Z',
+    'event.type': 'error',
+    eventID: '123456',
+    groupID: '1',
+    id: '98654',
+    location: 'main.js',
+    message: 'TestException',
+    platform: 'native',
+    projectID: '123',
+    tags: [{value: 'production', key: 'production'}],
+    title: 'TestException',
+  },
+  data: stacktrace,
+  stacktrace,
+  hasHierarchicalGrouping: false,
+  groupingCurrentLevel: undefined,
+  meta: undefined,
+};
+
+describe('ExceptionStacktraceContent', function () {
+  const organization = TestStubs.Organization();
+
+  it('default behaviour', function () {
+    const {container} = render(
+      <OrganizationContext.Provider value={organization}>
+        <ExceptionStacktraceContent {...props} />
+      </OrganizationContext.Provider>
+    );
+
+    expect(container).toSnapshot();
+  });
+
+  it('should return an emptyRender', function () {
+    render(
+      <OrganizationContext.Provider value={organization}>
+        <ExceptionStacktraceContent {...props} stacktrace={null} />
+      </OrganizationContext.Provider>
+    );
+
+    expect(screen.queryByText(frames[0].filename)).not.toBeInTheDocument();
+  });
+
+  it('should return the EmptyMessage component', function () {
+    render(
+      <OrganizationContext.Provider value={organization}>
+        <ExceptionStacktraceContent
+          {...props}
+          stackView={STACK_VIEW.APP}
+          chainedException={false}
+          stacktrace={{...stacktrace, frames: []}}
+        />
+      </OrganizationContext.Provider>
+    );
+
+    expect(
+      screen.getByText('No app only stack trace has been found!')
+    ).toBeInTheDocument();
+  });
+
+  it('should not return the EmptyMessage component', () => {
+    render(
+      <OrganizationContext.Provider value={organization}>
+        <ExceptionStacktraceContent
+          {...props}
+          stacktrace={{...stacktrace, frames: [{...frames[0], inApp: true}, frames[1]]}}
+        />
+      </OrganizationContext.Provider>
+    );
+    expect(
+      screen.getByText(textWithMarkupMatcher('foo/baz.c at line 1'))
+    ).toBeInTheDocument();
+  });
+
+  it('should render system frames if "stackView: app" and there are no inApp frames and is a chained exceptions', function () {
+    render(
+      <OrganizationContext.Provider value={organization}>
+        <ExceptionStacktraceContent
+          {...props}
+          stackView={STACK_VIEW.APP}
+          chainedException
+        />
+      </OrganizationContext.Provider>
+    );
+
+    // frame 1
+    expect(screen.getByText(frames[0].filename)).toBeInTheDocument();
+
+    // frame 2
+    expect(screen.getByText(frames[1].filename)).toBeInTheDocument();
```

Comment:
For tests like these, does it make sense to do something like

```
for(const frame of frames){
 expect(frame.filename).toBeInTheDocument()
}
```
or do we only expect it to show the first and 2nd element?